### PR TITLE
Contact me branch

### PR DESCRIPTION
**Pull request by suthish777**
My partner and I worked and committed to this project as a pair-programming exercise.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,6 +176,24 @@ <h2 class="about-heading-small">Skills</h2>
...
     </ul>
+    <!-- Contact Me Section Starts -->
+    <div id="contact" class="contact-me-form-container">
+        <h1 class="contact-me-form-heading">Contact me</h1>
+        <p class="contact-me-form-paragraph">If you have an application you are interested in developing, a
+            feature that you need built
+            or a project that
+            needs coding. I'd love to help with it</p>
+        <form action="https://formspree.io/f/mqkoaqjr" method="POST" class="contact-me-form">
+            <input class="contact-me-form-text-input" required maxlength="30" name="name" type="text"
```
Comment (joskalenda) [inline index.html]: - [ ] You have done what we `EXPECTED` to see, which was lovely and genius from you ❤️, but the minor issue is in the design above, please make sure you adjust the padding on the left and add the black line at the bottom of the page and request for a new review

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @suthish777 and @peteronyam , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Great design
- [ ].. ✅ Green linters
- [ ].... ✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @suthish777  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] You did a wonderful job styling your project :+1:, but as you can see below the background above the blue background should be white to match with the Figma template.

**Your design**

Screenshot from 2023-02-24 07-15-19

**Figma**

Screenshot from 2023-02-24 07-16-00


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — COMMENTED** [review]
Hi @peteronyam, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

From the [work and submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode) of the project instructions, you both are required to repeat the work in your own repository. Kindly reach out to @suthish777 so that you'll both implement the project in your repository.

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by adel-gu — COMMENTED** [review]
# Status: INVALID ❌ 

Hi @suthish777 👋🏼  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

From the [work and submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode) of the project instructions, you and your partner should collaborate in both **repos** and each one of you should ask for a code review with his own Pull Request 👍🏼, since @suthish777 PR is approved, now @peteronyam need's to ask for a code review using his project from his repo.

image





Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**